documentation/installing-az-modules.md

```diff
@@ -0,0 +1,69 @@
+# Installing `Az` Modules
+
+## Overview
+
+By default, modules are installed from the [PowerShell Gallery](https://www.powershellgallery.com/), which is the central repository for accessing published PowerShell modules (equivalent to NuGet for .NET, npm for JavaScript, etc.). With the [`Install-Module`](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module) cmdlet, users can specify which modules they want to install; in addition, users can provide the `-Repository` parameter to specify which repository they want to install modules from (if this parameter isn't provided, then the cmdlet defaults to using PowerShell Gallery).
+
+## Removing Modules
+
+In some cases, existing `Az` modules will need to be removed before a new version can be installed. Since the `Uninstall-Module` cmdlet does not currently remove modules and their dependencies, users will need to manually delete the folders where the modules were installed to.
+
+To figure out if you have any `Az` modules currently installed, as well as the location they are found, use the following command:
+
+```
+Get-Module -Name Az* -ListAvailable
+```
+
+This command will list all modules installed on your machine that are found in your `$env:PSModulePath`. Deleting the corresponding `Az.*` folders in the file explorer will remove these modules from your machine.
+
+## Registering Repositories
+
+In some cases, users will need to install modules from a different repository than the PowerShell Gallery -- this can be a new endpoint, or even a local folder containing `.nupkg` files. In either case, the [`Register-PSRepository`](https://docs.microsoft.com/en-us/powershell/module/powershellget/register-psrepository) cmdlet should be used to create a new local repository that can be used to install modules from.
```

Comment:
Should we mention anything about the test gallery? I find that, when interacting with RPs, we always need to explain how to register the test gallery and then install modules from it. Seems like a common enough scenario to mention.

Reply:
Added a snippet about registering the Test Gallery.